### ISSUE TITLE
Error when importing my own .pb file 

### ISSUE THREAD

**Issue by mdshopon** [llSourcell/A_Guide_to_Running_Tensorflow_Models_on_Android#3]
Hello there, 
I have trained a model of my own dataset which consists of Bangla characters of 60 different labels. When I replaced the opt_mnist_convnet-keras.pb with my own file I continiously get this
"Error When Intializing classifiers"
